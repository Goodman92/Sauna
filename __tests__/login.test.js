import React from 'react';
import renderer from 'react-test-renderer';

import LoginScreen from '../src/screens/login/login';


it('renders screen', () => {
  const tree = renderer.create(<LoginScreen/>).toJSON();

  expect(tree).toMatchSnapshot();
});


