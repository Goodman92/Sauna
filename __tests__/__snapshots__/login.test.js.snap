// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders screen 1`] = `
<View
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <Image
    source={
      Object {
        "process": [Function],
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": "100%",
          "width": "100%",
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "marginTop": 10,
        "paddingLeft": 10,
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "opacity": 1,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#fff",
              "fontSize": 20,
            },
            Object {
              "width": 20,
            },
            Object {
              "fontFamily": "FontAwesome",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }
      >
        ÔÅß
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#fff",
          }
        }
      >
         Create Account
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "column",
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "flex": 3,
            "justifyContent": "center",
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "flexDirection": "column",
              "width": 125,
            },
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#fff",
                "textAlign": "center",
              },
              Object {
                "fontSize": 24,
                "letterSpacing": 0,
              },
            ]
          }
        >
          LOGIN
        </Text>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "borderColor": "#fff",
                  "borderLeftWidth": 0,
                  "borderRadius": 1,
                  "borderRightWidth": 0,
                  "borderWidth": 0.5,
                  "flex": 3,
                  "height": 3,
                },
              ]
            }
          />
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#fff",
                  "flex": 2,
                  "textAlign": "center",
                },
              ]
            }
          >
            to
          </Text>
          <View
            style={
              Array [
                Object {
                  "borderColor": "#fff",
                  "borderLeftWidth": 0,
                  "borderRadius": 1,
                  "borderRightWidth": 0,
                  "borderWidth": 0.5,
                  "flex": 3,
                  "height": 3,
                },
              ]
            }
          />
        </View>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#fff",
                "textAlign": "center",
              },
              Object {
                "fontSize": 34,
                "letterSpacing": -0.5,
              },
            ]
          }
        >
          SAUNA
        </Text>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "flex": 6,
            "paddingLeft": 50,
            "paddingRight": 50,
          },
        ]
      }
    >
      <View
        onResponderRelease={[Function]}
        onStartShouldSetResponder={[Function]}
        pointerEvents="auto"
      >
        <View
          style={
            Object {
              "backgroundColor": "transparent",
              "borderBottomColor": "rgba(255, 255, 255, 1)",
              "borderBottomWidth": 0.5,
              "height": 64,
              "paddingBottom": 8,
              "paddingTop": 32,
            }
          }
        >
          <View
            style={
              Object {
                "position": "absolute",
                "top": 36,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              numberOfLines={1}
              style={
                Object {
                  "color": "rgba(255, 255, 255, 1)",
                  "fontSize": 16,
                }
              }
            >
              Email
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
              }
            }
          >
            <TextInput
              allowFontScaling={true}
              autoCapitalize="sentences"
              disableFullscreenUI={true}
              editable={true}
              onBlur={[Function]}
              onChange={[Function]}
              onChangeText={[Function]}
              onContentSizeChange={[Function]}
              onFocus={[Function]}
              selectionColor="#fff"
              style={
                Array [
                  Object {
                    "flex": 1,
                    "margin": 0,
                    "padding": 0,
                    "top": 2,
                  },
                  Object {
                    "color": "rgba(0, 0, 0, .87)",
                    "fontSize": 16,
                    "height": 24,
                    "textAlign": "left",
                  },
                  undefined,
                ]
              }
              underlineColorAndroid="transparent"
              value=""
            />
          </View>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "height": 8,
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "flex-start",
                  "bottom": 0,
                  "left": 0,
                  "paddingVertical": 4,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                numberOfLines={1}
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "color": "rgb(213, 0, 0)",
                    "fontSize": 0,
                    "opacity": 0,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "alignItems": "flex-start",
                  "bottom": 0,
                  "left": 0,
                  "paddingVertical": 4,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                numberOfLines={1}
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "color": "#fff",
                    "fontSize": 12,
                    "opacity": 1,
                  }
                }
              />
            </View>
          </View>
        </View>
      </View>
      <View
        onResponderRelease={[Function]}
        onStartShouldSetResponder={[Function]}
        pointerEvents="auto"
      >
        <View
          style={
            Object {
              "backgroundColor": "transparent",
              "borderBottomColor": "rgba(255, 255, 255, 1)",
              "borderBottomWidth": 0.5,
              "height": 64,
              "paddingBottom": 8,
              "paddingTop": 32,
            }
          }
        >
          <View
            style={
              Object {
                "position": "absolute",
                "top": 36,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              numberOfLines={1}
              style={
                Object {
                  "color": "rgba(255, 255, 255, 1)",
                  "fontSize": 16,
                }
              }
            >
              Password
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
              }
            }
          >
            <TextInput
              allowFontScaling={true}
              autoCapitalize="sentences"
              disableFullscreenUI={true}
              editable={true}
              onBlur={[Function]}
              onChange={[Function]}
              onChangeText={[Function]}
              onContentSizeChange={[Function]}
              onFocus={[Function]}
              selectionColor="#fff"
              style={
                Array [
                  Object {
                    "flex": 1,
                    "margin": 0,
                    "padding": 0,
                    "top": 2,
                  },
                  Object {
                    "color": "rgba(0, 0, 0, .87)",
                    "fontSize": 16,
                    "height": 24,
                    "textAlign": "left",
                  },
                  undefined,
                ]
              }
              underlineColorAndroid="transparent"
              value=""
            />
          </View>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "height": 8,
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "flex-start",
                  "bottom": 0,
                  "left": 0,
                  "paddingVertical": 4,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                numberOfLines={1}
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "color": "rgb(213, 0, 0)",
                    "fontSize": 0,
                    "opacity": 0,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "alignItems": "flex-start",
                  "bottom": 0,
                  "left": 0,
                  "paddingVertical": 4,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                numberOfLines={1}
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "color": "#fff",
                    "fontSize": 12,
                    "opacity": 1,
                  }
                }
              />
            </View>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "marginTop": 40,
          }
        }
      >
        <View
          accessible={true}
          onLayout={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          pointerEvents="box-only"
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderRadius": 2,
                "flexDirection": "row",
                "height": 36,
                "justifyContent": "center",
                "paddingHorizontal": 16,
              },
              false,
              Object {
                "backgroundColor": "#fff",
                "borderColor": "rgba(0,0,0,.12)",
              },
              false,
              false,
              Object {
                "shadowColor": "#000000",
                "shadowOffset": Object {
                  "height": 1,
                  "width": 0,
                },
                "shadowOpacity": 0.3,
                "shadowRadius": 1,
                "zIndex": 1,
              },
              undefined,
            ]
          }
          testID={null}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 14,
                  "fontWeight": "500",
                },
                false,
                undefined,
                false,
                false,
                undefined,
                undefined,
              ]
            }
          >
            LOGIN
          </Text>
          <View
            pointerEvents="none"
            style={
              Array [
                Object {
                  "bottom": 0,
                  "left": 0,
                  "overflow": "hidden",
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                },
                undefined,
              ]
            }
          >
            <View
              pointerEvents="none"
              style={
                Object {
                  "backgroundColor": "rgba(0, 0, 0, 0.87)",
                  "bottom": 0,
                  "left": 0,
                  "opacity": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "zIndex": 1,
                }
              }
            />
            <View
              pointerEvents="none"
              style={
                Object {
                  "backgroundColor": "rgba(0, 0, 0, 0.87)",
                  "borderRadius": 100,
                  "height": 200,
                  "left": -100,
                  "opacity": 0.12,
                  "position": "absolute",
                  "top": -100,
                  "transform": Array [
                    Object {
                      "scale": 0,
                    },
                  ],
                  "width": 200,
                  "zIndex": 1,
                }
              }
            />
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
              },
              Object {
                "marginTop": 5,
              },
              Object {
                "marginBottom": 5,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "borderColor": "#fff",
                  "borderLeftWidth": 0,
                  "borderRadius": 1,
                  "borderRightWidth": 0,
                  "borderTopWidth": 0,
                  "borderWidth": 0.5,
                  "flex": 5,
                  "height": 3,
                },
                Object {
                  "marginLeft": 40,
                },
              ]
            }
          />
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#fff",
                  "flex": 2,
                  "textAlign": "center",
                },
              ]
            }
          >
            or
          </Text>
          <View
            style={
              Array [
                Object {
                  "borderColor": "#fff",
                  "borderLeftWidth": 0,
                  "borderRadius": 1,
                  "borderRightWidth": 0,
                  "borderTopWidth": 0,
                  "borderWidth": 0.5,
                  "flex": 5,
                  "height": 3,
                },
                Object {
                  "marginRight": 40,
                },
              ]
            }
          />
        </View>
        <View
          accessible={true}
          onLayout={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          pointerEvents="box-only"
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderRadius": 2,
                "flexDirection": "row",
                "height": 36,
                "justifyContent": "center",
                "paddingHorizontal": 16,
              },
              false,
              Object {
                "backgroundColor": "#fff",
                "borderColor": "rgba(0,0,0,.12)",
              },
              false,
              false,
              Object {
                "shadowColor": "#000000",
                "shadowOffset": Object {
                  "height": 1,
                  "width": 0,
                },
                "shadowOpacity": 0.3,
                "shadowRadius": 1,
                "zIndex": 1,
              },
              Object {
                "backgroundColor": "#3b5998",
              },
            ]
          }
          testID={null}
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#fff",
                  "fontSize": 20,
                },
                Object {
                  "width": 20,
                },
                Object {
                  "fontFamily": "FontAwesome",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            ÔÇö
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 14,
                  "fontWeight": "500",
                },
                false,
                undefined,
                false,
                false,
                undefined,
                Object {
                  "color": "#fff",
                },
              ]
            }
          >
            LOGIN WITH FACEBOOK
          </Text>
          <View
            pointerEvents="none"
            style={
              Array [
                Object {
                  "bottom": 0,
                  "left": 0,
                  "overflow": "hidden",
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                },
                undefined,
              ]
            }
          >
            <View
              pointerEvents="none"
              style={
                Object {
                  "backgroundColor": "rgba(0, 0, 0, 0.87)",
                  "bottom": 0,
                  "left": 0,
                  "opacity": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "zIndex": 1,
                }
              }
            />
            <View
              pointerEvents="none"
              style={
                Object {
                  "backgroundColor": "rgba(0, 0, 0, 0.87)",
                  "borderRadius": 100,
                  "height": 200,
                  "left": -100,
                  "opacity": 0.12,
                  "position": "absolute",
                  "top": -100,
                  "transform": Array [
                    Object {
                      "scale": 0,
                    },
                  ],
                  "width": 200,
                  "zIndex": 1,
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
